src/styles/stars.scss

```diff
@@ -0,0 +1,32 @@
+.stars {
+  gap: 4px;
+  margin-top: 16px;
+  display: flex;
+  margin-left: 16px;
+  margin-bottom: 24px;
+
+  &__star {
+    background-image: url(/src/images/star.svg);
```

Comment:
Verify that the image path `/src/images/star.svg` is correct and the image is accessible.